src/emulator/main.cpp

```diff
@@ -28,9 +28,7 @@
 #include <util/log.h>
 #include <util/string_utils.h>
 
-#ifdef USE_GDBSTUB
-#include <gdbstub/functions.h>
-#endif
+#include <SDL.h>
```

Comment:
Is this a leftover from rebase or needed

Reply:
Rebase 😦  Fixed in new commit.